src/types.ts

```diff
@@ -115,7 +115,7 @@ export type BlockchainCollectionV2Asset = BaseBlockchainAsset & {
 /**
  * @public
  */
- export type BlockchainCollectionThirdParty = BaseBlockchainAsset & {
+export type BlockchainCollectionThirdPartyItem = BaseBlockchainAsset & {
   namespace: 'decentraland'
   type: "blockchain-collection-third-party"
```

Comment:
id should be different for third party collection and third party collection item

Reply:
That's something that I wasn't sure about, anyway, should I make a breaking change for the items? Or how should I name the collection itself? 🤔

Reply:
To be consistent with the others collection, i think it should be  `BlockchainCollectionThirdParty` for the collection and `BlockchainCollectionThirdPartyAsset` for the items.

Reply:
I thought the same, but doing that will imply a breaking change: `BlockchainCollectionThirdParty` ->  `BlockchainCollectionThirdPartyAsset` and we want to avoid that

Reply:
:sob: